1_fetch/src/get_sb_data.R

```diff
@@ -258,11 +258,31 @@ prep_feature_vars <- function(sb_var_data, sites, retain_vars) {
     select(COMID, label, avg_annual) %>%
     pivot_wider(names_from = "label", values_from = "avg_annual")
   
+  #determine dominant physiographic region
+  phys_region <- data %>%
+    select(COMID, contains("_PHYSIO_")) %>%
+    select(-contains("AREA")) %>%
+    group_by(COMID) %>%
+    summarise_all(mean) %>%
+    pivot_longer(!COMID, names_to = "name", values_to = "value") %>%
+    mutate(unit = str_sub(name, 1, 3), 
+           region = str_sub(name, 12)) %>%
+    group_by(COMID, unit) %>%
+    summarise(dom_phys_reg_ind = which.max(value), .groups = "drop") %>%
```

Comment:
Have you handled the case when more than one region has the same maximum coverage area?

Reply:
I have not... it appears that `which.max()` (and all `which()` functions) default to the smallest index when there are ties. Not sure how we would choose between one or the other in our case, will hold off for today's meeting.

Reply:
We ran out of time to discuss this today. There aren't any ties in Gages2.1 so it isn't technically an issue for the moment, but there could be COMIDs in our prediction space with ties. I'm thinking if there were to be a tie in either `ACC_` or `TOT_` regions, we could use the dominant `CAT_` region as the tiebreaker (defer to the physical location of the reach rather than upstream watershed area). I would be surprised if there were any ties in `CAT_` but I suppose it is possible... thoughts?

Reply:
I agree it would be less likely for CAT to have ties. That sounds like a good approach to me